deploy/charts/trust-manager/templates/certificate.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if not .Values.app.webhook.tls.helmCert.enabled -}}
+{{- if .Values.app.webhook.tls.certManager.enabled -}}
```

Comment:
This appears like a breaking change. It it intentional? If possible, I think we should still support the `app.webhook.tls.helmCert.enabled` Helm value.

Reply:
the `app.webhook.tls.helmCert.enabled` still works, but for for existing installation, the `app.webhook.tls.certManager.enabled` var would have to be set to false, which indeed is a breaking change.

Should we have instead a `app.webhook.tls.externalProvider.enabled=false` var that when enabled, disables the cert-manager certficates creation ?

Reply:
I am not sure if this will work, but could it be an option to add a new optional Helm value to set the name of the webhook certificate secret? And when user sets this value, all use of cert-manager or Helm to generate a webhook certificate is turned off?